website/src/lib/chat_stream.ts

```diff
@@ -39,8 +48,15 @@ export async function handleChatEventStream({
         // handle error
         await onError(chunk.error);
         return chunk.message;
+      } else if (chunk.event_type === "plugin_intermediate") {
+        await onPluginIntermediateResponse({
+          currentPluginThought: chunk.current_plugin_thought,
+          currentPluginAction: chunk.current_plugin_action_taken,
+          currentPluginActionResponse: chunk.current_plugin_action_response,
+          currentPluginActionInput: chunk.current_plugin_action_input,
+        });
       } else {
-        console.error("Unexpected event", chunk);
+        console.log("Unexpected event", chunk);
```

Comment:
Why do we need to change to log? I think error will make more sense here

Reply:
Is probably my bad merge, I don't remember changing it to log... will revert it to error.